run-build.sh

```diff
@@ -93,11 +93,18 @@ while [[ $# > 0 ]]; do
             args=( "${args[@]/$2}" )
             shift
             ;;
+        --env-vars)
```

Comment:
Conventionally, the env-vars vs. argument style is either:

```sh
./build.sh --runtime-id ubuntu.14.04-arm -c Release

# or
DOTNET_RUNTIME_ID=ubuntu.14.04-arm  ./build.sh -c Release
# windows powershell equivalent of:
$env:DOTNET_RUNTIME_ID=ubuntu.14.04-arm; ./build.ps1 -c Release
```

In case both are supported and provided, command-line argument will take the precedence.